src/tests/ftest/scripts/main.sh

```diff
@@ -256,9 +256,18 @@ fi
 # shellcheck disable=SC2086
 export WITH_VALGRIND
 export STAGE_NAME
+
+launch_node_args = "-ts \"${TEST_NODES}\""
+if [ "${STAGE_NAME}" == "Functional Hardware Xlarge" ]; then
```

Comment:
(lint) Couldn't find 'fi' for this 'if'. [SC1046]
(lint) Couldn't parse this if expression. Fix to allow more checks. [SC1073]